test/e2e/es/version_upgrade_test.go

```diff
@@ -182,13 +185,22 @@ func TestVersionUpgradeSingleToLatest8x(t *testing.T) {
 
 	test.SkipInvalidUpgrade(t, srcVersion, dstVersion)
 
+	resources := corev1.ResourceRequirements{
```

Comment:
Maybe add a comment explaining your motivation for adding the explicit resource requirement.